Source/AudioVisualizer.cpp

```
@@ -615,62 +706,151 @@ void AudioVisualizer::renderCompactMeter(NUIRenderer& renderer) {
-    renderer.fillRoundedRect(bounds, 4, backgroundColor_.darkened(0.2f));
-    
-    // Aggressive scaling for better visibility
-    // Most music peaks around 0.1-0.3 RMS, so we scale it to fill the meter
-    float scaledLevel = std::min(1.0f, level * 5.0f); // Increased scaling from 3.0 to 5.0
-    float levelHeight = scaledLevel * bounds.height;
-    
-    // Draw the level bar (colored - cyan/magenta)
-    if (levelHeight > 1.0f) { // Only draw if visible
-        NUIRect levelRect(bounds.x, bounds.y + bounds.height - levelHeight, bounds.width, levelHeight);
-        renderer.fillRoundedRect(levelRect, 4, color);
+    renderer.fillRoundedRect(bounds, 3.0f, backgroundColor_.darkened(0.25f));
```

Clang (14.0.6)

[warning] 3.0f is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)